src/sections/Project-grid/index.js

```diff
@@ -18,62 +18,51 @@ const ProjectPage = ({data}) => {
             <PageHeader title="Projects" path="Projects"/>
             <div className="project-page-wrapper">
                 <Container>
-                    <Row>
-                        <Col xs={12} sm={6} lg={4}>
-                            <Link to="/meshery">
-                                <div className="project__block__inner">
-                                    <img src={meshery}
-                                        alt="Meshery"
-                                    />
-                                    <h5>Meshery</h5>
-                                </div>
-                            </Link>
-                        </Col>
-                        <Col xs={12} sm={6} lg={4}>
-                            <Link to="/landscape">
-                                <div className="project__block__inner">
-                                    <img src={landscape}
-                                        alt="Landscape"
-                                    />
-                                    <h5>Landscape</h5>
-                                </div>
-                            </Link>
-                        </Col>
-                        <Col xs={12} sm={6} lg={4}>
-                            <Link to="/projects/image-hub">
-                                <div className="project__block__inner">
-                                    <img src={imagehub}
-                                        alt="Image Hub"
-                                    />
-                                    <h5>Image Hub</h5>
-                                </div>
-                            </Link>
-                        </Col>
-                        <Col xs={12} sm={6} lg={4}>
-                            <Link to="/projects/service-mesh-interface-conformance">
-                                <div className="project__block__inner">
-                                    <img src={smi}
-                                        alt="Service Mesh Interface Conformance" 
+                    <div className="project__grid">
+                        {data.allMdx.nodes.map(({id, frontmatter, fields }) => (
+                            <Link to={fields.slug} key={id} className={`project__card ${frontmatter.title === "Service Mesh Performance" ? "five": "six"}`}>
```

Comment:
Can you separate these out like the other @bartkalanski? We are going to create separate pages for them

Reply:
Yes no problem!